packages/grafana-runtime/src/config.ts

```diff
@@ -99,6 +99,10 @@ export class GrafanaBootConfig implements GrafanaConfig {
     enabled: false,
   };
 
+  validatedQueries = {
```

Comment:
what is this?

Reply:
We were following what appeared to be existing paradigms for setting feature flags. Is this unnecessary?